contrib/meson.build

```diff
@@ -12,6 +12,10 @@ configure_file(
   configuration: con2,
 )
 
+if get_option('tests')
```

Comment:
Maybe only when `get_option('tests') and get_option('b_sanitize')`?

Reply:
Nah, I don't really want to conditionalize what gets installed on compiler options -- we'd have to make the export conditional and it would make debugging a whole lot harder to save 4kb in a subpackage we don't install by default anyway.

Reply:
OK, sounds fine then.